scripts/packaging/publish_release.py

```diff
@@ -120,9 +120,7 @@
                         except requests.exceptions.ConnectionError:
                             remainingTrials -= 1
                             print('Release upload failed (remaining trials: %d)' % remainingTrials)
-                    if (releaseExists and
-                            not (options.tag and not options.tag.startswith('refs/heads/')) and
-                            not releaseCommentModified and
+                    if (releaseExists and tag.startswith('nightly_') and not releaseCommentModified and
```

Comment:
```suggestion
                    if (releaseExists and not release.draft and not releaseCommentModified and
```
I am not completely sure whether it is going to work. But I believe it is better to define a logic (whether it is a draft or not) in the single place.